source/javascripts/controllers/_StackController.js.erb

```diff
@@ -163,9 +163,16 @@
 				if (stack.descriptionURLGen2 !== undefined) {
 					return stack.descriptionURLGen2;
 				}
-				return stack.descriptionURL;
+
+				if (stack.descriptionURL) {
+					return stack.descriptionURL;
+				}
 			};
 
+			viewModel.hasDescriptionURLForStack = function(stack) {
+				return stack.descriptionURLGen2 || stack.descriptionURL;
```

Comment:
`"a" || "b"` would return `"a"`, whereas a function called `hasDescriptionURLForStack` should return a boolean. Cast the result to that:
```suggestion
				return !!(stack.descriptionURLGen2 || stack.descriptionURL);
```